circle.yml

```diff
@@ -0,0 +1,26 @@
+dependencies:
+  pre:
+    # Replace default bundler version, which is a bit older
+    - yes y | gem uninstall bundler
+    - gem install bundler --pre
+  post:
+    - npm run build
+
+deployment:
+  staging:
+    branch: master
+    heroku:
+      appname: cs1332qna
+
+machine:
+  # Ruby version specified in .ruby-version
```

Comment:
FYI, this particular comment is just referring to the fact that CircleCI can also detect the ruby version from `.ruby-version` instead of having a config section here like for node.

I at least am using `rbenv`; I don't know about anyone else.